src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -1,34 +1,64 @@
 package edu.kis.vh.nursery;
 
+/**
+ * DefaultCountingOutRhymer is counting-out rhyme used to conducting complicated number games
+ */
 public class DefaultCountingOutRhymer {
 
-    private int[] numebrs = new int[12];
+    private static final int CAPACITY = 12;
+    private static final int STACK_EMPTY_VALUE = -1;
+    private static final int STACK_FULL_VALUE = 11;

```

Comment:
This constant is related to CAPACITY. Changing only one leads to an error (inconsistency).